Java Course/src/randomProgramy/Program.java

```diff
@@ -67,22 +68,46 @@ public static void out(Integer number) {
 	 * 
 	 * @param list
 	 * 
-	 * @return Zwraca przednajwi�ksz� liczb� z listy, lub null je�eli przednajwi�ksza liczba nie istnieje
+	 * @return Zwraca przednajwi�ksz� liczb� z listy, lub null je�eli
+	 *         przednajwi�ksza liczba nie istnieje
 	 */
 	public static Integer highestButOne(List<Integer> list) {
 		Collections.sort(list);
 		Collections.reverse(list);
 		int i;
-		if (list.size() == 2)
-			return list.get(1);
-		else {
+		if (list.size() == 2) {
+			if (list.get(0) != list.get(1))

```

Comment:
Łukasz!!!!!! tak nie wolno!!!!!!
list jest typu: List<Integer> a co to oznacza, proponuję pogadać o tym przez telefon.